src/browser/modules/Stream/SysInfoFrame/SysInfoFrame.tsx

```diff
@@ -169,7 +169,7 @@ export class SysInfoFrame extends Component<
     if (bus && isConnected && useDb) {
       this.setState({ lastFetch: Date.now() })
       bus.self(
-        CYPHER_REQUEST,
+        ROUTED_CYPHER_READ_REQUEST,
```

Comment:
It might make sense to use write here to make sure it goes to the leader, which usually has the stats we care about.

Reply:
We do a routed -> If we're on a cluster we say stats are from the LEADER server. We could say browser only supports listing sysinfo from leader, other wise use opts mgmr